.github/workflows/generate-config.yaml

```
@@ -0,0 +1,133 @@
+    types: [closed]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-config:
+    runs-on: ubuntu-latest
+
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      CHAINS_PAT: ${{ secrets.CHAINS_PAT }}
+      EXTERNAL_REPO: ${{ secrets.EXTERNAL_REPO }}
+
+    steps:
+      - name: Checkout axelar-contract-deployments repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Check if PR was merged
+        id: check_if_merged
+        run: |
+          if [ "${{ github.event.pull_request.merged }}" != "true" ]; then
+            echo "PR was not merged. Exiting."
+            exit 0
+          fi
+
+      - name: Get the merge commit SHA
+        id: merge_commit
+        run: |
+          MERGE_COMMIT=$(git log -1 --pretty=format:"%H")
+          echo "MERGE_COMMIT=$MERGE_COMMIT" >> $GITHUB_ENV
+
+      - name: Determine NETWORKS
+        id: determine_networks
+        run: |
+          echo "Checking modified files in commit ${{ env.MERGE_COMMIT }}"
+          MODIFIED_FILES=$(git diff --name-only HEAD~1..HEAD)
+          echo "Modified files: $MODIFIED_FILES"
+
+          NETWORKS=()
+
+          if echo "$MODIFIED_FILES" | grep -q "axelar-chains-config/info/mainnet.json"; then
+            NETWORKS+=("mainnet")
+          fi
+          if echo "$MODIFIED_FILES" | grep -q "axelar-chains-config/info/testnet.json"; then
+            NETWORKS+=("testnet")
+          fi
+          if echo "$MODIFIED_FILES" | grep -q "axelar-chains-config/info/stagenet.json"; then
+            NETWORKS+=("stagenet")
+          fi
+          if echo "$MODIFIED_FILES" | grep -q "axelar-chains-config/info/devnet-amplifier.json"; then
+            NETWORKS+=("devnet-amplifier")
+          fi
+
+          if [ ${#NETWORKS[@]} -eq 0 ]; then
+            echo "No relevant changes detected."
+            echo "NETWORKS=none" >> $GITHUB_ENV
+            exit 0
+          fi
+
+          echo "Detected networks: ${NETWORKS[*]}"
+          echo "NETWORKS=${NETWORKS[*]}" >> $GITHUB_ENV
+
+      - name: Checkout external repository
+        if: env.NETWORKS != 'none'
+        uses: actions/checkout@v4
+        with:
+          repository: axelarnetwork/${{ secrets.EXTERNAL_REPO }}
+          path: ${{ secrets.EXTERNAL_REPO }}
+          token: ${{ secrets.CHAINS_PAT }}
+
+      - name: Install jq
+        if: env.NETWORKS != 'none'
+        run: sudo apt-get install -y jq
+
+      - name: Setup Branch Name
+        if: env.NETWORKS != 'none'
+        id: branch-name
+        run: echo "branch=feat/config-update-$(date +'%Y-%m-%d-%H-%M-%S')" >> "$GITHUB_OUTPUT"
+
+      - name: Create branch and generate config
+        if: env.NETWORKS != 'none'
+        run: |
+          cd ${{ secrets.EXTERNAL_REPO }}
+          curl -sSL https://install.python-poetry.org | python3 -
+          poetry install --no-interaction
+          git config --local user.email "devops@axelar.network"
```

```suggestion
          git config --local user.email "cicd@interoplabs.io"
```
is there a cicd mailing list instead?

---

@milapsheth I don't have access to the mailing list management, and I just tested it, it doesn't seem I'm a part of that mailing list if it exists, can you double check? but I'm a part of `devops@interoplabs.io`.